CMakeLists.txt

```diff
@@ -309,6 +309,14 @@ include_directories(SYSTEM ${LIBUV_INCLUDE_DIRS})
 find_package(Msgpack 1.0.0 REQUIRED)
 include_directories(SYSTEM ${MSGPACK_INCLUDE_DIRS})
 
+option(PREFER_LUAJIT "Prefer LuaJIT over Lua when compiling executable. Test library always uses luajit." ON)
+
+find_package(LuaJit REQUIRED)
```

Comment:
I tried to just switch the nightlies to LuaJIT, but unfortunately LuaJIT is not supported on ARM64. Opened https://github.com/neovim/neovim/pull/6736.